protocols/aavegotchi/index.json

```diff
@@ -0,0 +1,33 @@
+{ 
+	"cname": "aavegotchi",
+	"name": "Aavegotchi",
+	"description": "",
+	"path": "aavegotchi",
```

Comment:
We should use the same path they use for Snapshot IMO, unless they requested otherwise https://snapshot.org/#/aavegotchi.eth

Reply:
Aah, ya definitely. I renamed the folder as well.